package.json

```diff
@@ -1,97 +1,106 @@
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.4",
-    "gulp-sass": "^2.2.0",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-stylestats": "^1.2.1",
-    "gulp-tslint": "^8.1.1",
-    "gulp-typescript": "2.13.0",
-    "gulp-util": "^3.0.7",
-    "gulp-zip": "^3.1.0",
-    "ip": "^1.1.5",
-    "jasmine-core": "~2.5.2",
-    "karma": "^1.6.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-jasmine": "^1.1.0",
-    "karma-mocha-reporter": "^2.2.3",
-    "merge-stream": "^1.0.0",
-    "prismjs": "^1.5.1",
-    "require-dir": "^0.3.0",
-    "rollup": "^0.37.0",
-    "rollup-plugin-buble": "^0.15.0",
-    "rollup-plugin-commonjs": "^7.0.0",
-    "rollup-plugin-node-resolve": "^2.0.0",
-    "rollup-plugin-uglify": "^1.0.1",
-    "run-sequence": "^1.1.5",
-    "shelljs": "^0.7.8",
-    "systemjs-builder": "^0.16.9",
-    "through2": "^2.0.1",
-    "tslint": "^5.5.0",
-    "typescript": "~2.2.0"
-  },
-  "scripts": {
-    "start": "gulp serve",
-    "build": "gulp build --prod && gulp karma",
-    "build:dev": "gulp build",
-    "clang": "gulp clang:check",
-    "test": "gulp test",
-    "test:watch": "gulp test:watch",
-    "tslint": "gulp tslint:check",
-    "prepublish:all": "gulp npm:prepare --version $npm_package_version",
-    "publish:all": "npm publish dist/npm/clarity-ui; npm publish dist/npm/clarity-angular; npm publish dist/npm/clarity-icons",
-    "presample-app": "npm i dist/npm/clarity-ui; npm i dist/npm/clarity-icons; npm i dist/npm/clarity-angular;",
-    "sample-app": "gulp sample",
-    "preks:publish": "gulp npm:prepare --version $npm_package_version",
-    "ks:publish": "cd src/ks-app; ng build --preserve-symlinks --prod; cd ../../dist/ks-app; cat index.html > 404.html; surge; cd ../../",
-    "cssTest": "./docker-ctrl/cdc.js -t",
-    "cssUpdate": "./docker-ctrl/cdc.js -u"
-  }
+    "name": "clarity-all",
+    "version": "0.10.6",
+    "private": true,
+    "scripts": {
+        "start": "webpack-dev-server --port=4200 --host 0.0.0.0",
+        "clean": "del ./dist",
+        "build": "npm-run-all clean build:angular webpack build:icons",
+        "pretest": "webpack",
+        "test": "karma start ./karma.conf.js",
+        "test:travis": "npm-run-all clang:check tslint:check test",
+        "clang:check": "./scripts/clang-check.sh",
+        "clang:format": "node_modules/.bin/clang-format -style=file -i --glob=src/**/*.ts",
+        "prebuild:angular": "cp ./npm/clarity-angular/package.json ./src/clarity-angular/; replace '@VERSION' $npm_package_version ./src/clarity-angular/package.json",
+        "build:angular": "ng-packagr -p ng-package.json",
+        "postbuild:angular": "cd dist/clarity-angular; npm pack;",
+        "build:icons": "webpack --config webpack.icons.config.js",
+        "postbuild:icons": "node ./scripts/clarity-icons-svg.js ",
+        "npm:publish": "npm publish dist/clarity-ui; npm publish dist/clarity-angular/clarity-angular-$npm_package_version.tgz; npm publish dist/clarity-icons",
```

Comment:
we don't need prepublish now because npm run build does that. is that right?

Reply:
yes, that's correct.